src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -258,6 +264,30 @@ public void start(Promise<Void> startFuture) {
 
   }
 
+  private void startHealthCheckService() {
+    HttpServer httpServer = vertx.createHttpServer();
+
+    httpServer.requestHandler(request -> {
+      log.debug("path : {}", request.path());
+      if (request.path().equals(HEALTH_CHECK_PATH)) {
+        HttpServerResponse response = request.response();
+        response.setStatusCode(200);
+        response.putHeader("Content-Type", "text/plain");
+        response.end("OK");
+        log.debug("statusCode : {}", response.getStatusCode());
+        log.info("message : {}", response.getStatusMessage());
```

Comment:
This log message should be changed to a more informative message like "admin health check service response message : {}

Reply:
Changed the logger message.